www/class/centreonGraphStatus.class.php

```diff
@@ -209,6 +204,33 @@ protected function getOptions()
         }
         return $result;
     }
+
+    /**
+     * Get the RRDCacheD options of local RRD Broker
+     *
+     * @return array of RRDCacheD options
+     */
+    protected function getRrdCachedOptions()
+    {
+        $result = $this->pearDB->query("SELECT config_key, config_value
+            FROM cfg_centreonbroker_info AS cbi
+            INNER JOIN cfg_centreonbroker AS cb ON (cb.config_id = cbi.config_id)
+            INNER JOIN nagios_server AS ns ON (ns.id = cb.ns_nagios_server)
+            WHERE ns.localhost = '1'
+            AND cbi.config_key IN ('rrd_cached_option', 'rrd_cached')"
+        );
+
+        $rrdCachedOptions = [];
+        while ($row = $result->fetch()) {
+            if ($row['config_key'] === 'rrd_cached_option') {
```

Comment:
could avoid if/elsif :
$this->rrdCachedOptions[$row['config_key']] = $row['config_value'];